xrpl/models/amounts/mpt_amount.py

```
@@ -49,3 +49,16 @@ def to_currency(self: Self) -> MPTCurrency:
             The MPTCurrency for this MPTAmount.
         """
         return MPTCurrency(mpt_issuance_id=self.mpt_issuance_id)
+
+
+class MPTIssue:
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)